synchronizer/default_l1processors.go

```diff
@@ -26,6 +26,6 @@ func defaultsL1EventProcessors(sync *ClientSynchronizer, l2Blockchecker *actions
 	p.Register(actions.NewCheckL2BlockDecorator(elderberry.NewProcessorL1SequenceBatchesElderberry(sequenceBatchesProcessor, sync.state), l2Blockchecker))
 	// intialSequence is process in ETROG by the same class, this is just a wrapper to pass directly to ETROG
 	p.Register(elderberry.NewProcessorL1InitialSequenceBatchesElderberry(sequenceBatchesProcessor))
-	p.Register(feijoa.NewProcessorSequenceBlobs(sync.state))
+	p.Register(feijoa.NewProcessorSequenceBlobs(sync.state, sync.state, nil))
```

Comment:
why sync.state is needed twice? Could we remove the nil param?? If it is not used.Why is the purpose of keeping it?

Reply:
Because one is the objected related to Database and the other one is the object related to Executor. That is relevant in tests and also, from the point of view of this class is not assuming that both parts are implemented by the same object